tests/tests_manifests/data/opensearch-2.18.0-test.yml

```diff
@@ -0,0 +1,152 @@
+---
+schema-version: '1.0'
+name: OpenSearch
+ci:
+  image:
+    name: opensearchstaging/ci-runner:ci-runner-al2-opensearch-build-v1
+    args: -e JAVA_HOME=/opt/java/openjdk-21
+components:
+  - name: opensearch
+    smoke-test:
+      test-spec:
+        - opensearch.yml
```

Comment:
Where is this file located here?

Reply:
I'm thinking to have this file stored in a new spec folder inside the smoke tests workflow directory. Open about any other suggestions.